packages/frontend/app/lib/validate.server.ts

```diff
@@ -88,6 +88,7 @@ export const createAssetSchema = z
     code: z
       .string()
       .length(3, { message: 'Code should be 3 characters long' })
+      .max(6, { message: 'Maximum length of Code is 6 characters' })
```

Comment:
Hi @Dhoni77 , thank you for your contribution!
Could you please change the line before (90) to have a minimum of 3 characters?

Reply:
sure @sabineschaller